README.rst

```diff
@@ -14,34 +14,38 @@ Installation
 ------------
 
 You will need to obtain several different pieces of information
-in order to get this project up and running. Secrets will be stored in the
-``secrets`` directory, which is not versioned with Git. Non-secret settings
+in order to get this project up and running. Secret and non-secret settings
 will be stored in environment variables. In order to make it easier to get
 started, you can copy ``.env.example`` to ``.env``.
 
 Django
 ~~~~~~
-Create a secret key for Django, and store it in the file
-``secrets/django-secret-key``. You can run this code to do so:
+Create a secret key for Django, and store it in the ``.env`` file as SECRET_KEY.
+You can run this code to create a key:
 
 .. code-block:: bash
 
-    head -c 50 /dev/urandom > secrets/django-secret-key
+    head -c 50 /dev/urandom
```

Comment:
This produces a sequence of random bytes. Maybe we should add a `| base64` to this to get something they can paste into `.env`